src/test/scenarios/synapse/output/coredefault/src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -88,7 +88,7 @@ cffi==1.14.0
 chardet==3.0.4
 colorama==0.4.1
 ConfigArgParse==0.14.0
-cryptography==3.2
+cryptography==2.8
```

Comment:
I believe 3.2 is correct version. Could you merge the latest master?

Reply:
yes, will merge and regenerate test

Reply:
Hi @qiaozha , updated